docs/reporting/response-codes.asciidoc

```diff
@@ -0,0 +1,23 @@
+The Reporting APIs use HTTP response codes to indicate to give feedback. In automation,
+this helps external systems track the various possible job states:
+
+- **`200` (OK)**: As expected, Kibana returns `200` status in the response for
+  successful requests to queue or download reports.
++
+NOTE: Kibana will send a `200` response status for successfully queuing a Reporting job via
+the POST URL. This is true even if the job somehow fails later, since report
+generation happens asynchronously from queuing.
+
+- **`400` (Bad Request)**: When sending requests to the POST URL, if you don't use
+  `POST` as the HTTP method, or if your request is missing the `kbn-version` header,
+  Kibana will return a code `400` status response for the request.
+
+- **`503` (Service Unavailable)**: When using the `path` to request the download, you will get
+  a `503` status response if report generation hasn't completed yet. In that case,
+  retry after the number of seconds in the `Retry-After` header in the download API
+  response until the report is complete.
+
+- **`500` (Internal Server Error)**: When using the `path` to request the download, you
+  will get a `500` status response if the report isn't available due to an error when
+  generating the report. More information will be available in the Reporting management
```

Comment:
Suggest shortening this to:

More information is available at **Management > Kibana > Reporting**.  (note ending period)